src/com/oltpbenchmark/benchmarks/tpch/queries/Q1.java

```diff
@@ -35,7 +35,7 @@
             + "from "
             +     "lineitem "
             + "where "
-            +     "l_shipdate <= date '1998-12-01' - interval '95' day "
+            +     "l_shipdate <= ADDDATE('1998-12-01', interval -'95' day) "
```

Comment:
Can you remove all of the changes that you made to the TPC-H queries for Cubrid? We will have to fix the benchmark later to add real dialect files.

Reply:
Sure, we will remote these.